### ISSUE TITLE
Test: Sum of the money in the application is always zero

### ISSUE THREAD

**Issue by matthiasbeyer** [cashctrl/cashctrl#67]
I don't know how to write a test for this:

If you take all ledgers and sum up the amount of money, the sum is always equal to zero.

@TheNeikos maybe you can teach me how to write a test for this?

**Comment by TheNeikos** [comment]
Steps are:

1. Create two or three ledgers
2. Create a few transactions
3. Map over ledgers and [pluck](http://apidock.com/rails/ActiveRecord/Calculations/pluck) their value
4. Fold over that array with something like `&:+`
5. Check that it's 0 :dancer:

**Comment by matthiasbeyer** [comment]
@TheNeikos Yep. I don't know how to write the FactoryGirl code for creating ledgers and transactions, though... so ... we might tackle this step this weekend?

**Comment by matthiasbeyer** [comment]
Belongs to #52
